RM ANOVA: added effect sizes

raviselker opened the pull request:
Please forgive me the ugliness of the code (I'm aiming at functionality at the moment, I'll clean up and improve the speed of the code asap)

raviselker committed:
RM ANOVA: added effect sizes

raviselker committed:
RM ANOVA: added levene's test

raviselker committed:
RM ANOVA: added sphericity corrections

jonathon-love commented on R/anovarm.b.R:
```diff
@@ -112,10 +113,15 @@ anovaRMClass <- R6::R6Class(
+            
+            if (length(self$options$bs) == 0) {
+                for (var in rmVars)
+                    leveneTable$addFootnote(rowKey=var, "name", "Because there are no between subjects factors specified this assumption is always met.")
```
shouldn't begin a sentence with 'Because'. you can use *As* in it's place; `As there are ...`

is it that the assumption is always met, or is it that it simply doesn't apply?

raviselker replied:
It is met because there is only one group. You could also say that it does not apply but that requires more insight from the user I think.

jonathon-love replied:
> It is met because there is only one group

i don't think that's true, but you're right about the alternative requiring more insight from the user, so i'm happy with it as is.

raviselker replied:
There's always equality of variances if there is only one group, so technically the assumption is met.

jonathon-love replied:
yeah, i guess that's true.

raviselker committed:
RM ANOVA: tidy up

raviselker committed:
Reliability Analysis: Fixed small error

The analyses didn't behave well when variable names had spaces in them.
This commit fixes that.

raviselker committed:
RM ANOVA: added 'running' status to slow tables
